files/common/usr/bin/get-appliance-version

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 by Delphix. All rights reserved.
```

Comment:
This copyright should be updated to use the Apache format, which is different than what the app-gate uses (which I believe this is copied from, right?).

Reply:
Thanks, yes, fixed it to follow similar to other scripts (like `get-property-from-image`).